cms/djangoapps/contentstore/tests/test_course_settings.py

```diff
@@ -139,113 +139,103 @@ def test_marketing_site_fetch(self):
             self.assertNotContains(response, "Course Introduction Video")
             self.assertNotContains(response, "Requirements")
 
-    def _seed_milestone_relationship_types(self):
-        """
-        Helper method to prepopulate MRTs so the tests can run
-        Note the settings check -- exams feature must be enabled for the tests to run correctly
-        """
-        if settings.FEATURES.get('ENTRANCE_EXAMS', False):
-            from milestones.models import MilestoneRelationshipType
-            MilestoneRelationshipType.objects.create(name='requires')
-            MilestoneRelationshipType.objects.create(name='fulfills')
-
-    @patch.dict(settings.FEATURES, {'ENTRANCE_EXAMS': True})
     def test_entrance_exam_created_updated_and_deleted_successfully(self):
```

Comment:
If this test only runs when `ENTRANCE_EXAMS` is turned on, can you do that by using the skip facility in unittest? (https://docs.python.org/2/library/unittest.html#unittest.skip)